### ISSUE TITLE
Create "How to run CSSclasses in your city" page

### ISSUE THREAD

**Issue by lukaszklis** [CSSclasses/CSSclasses#28]
It would be great to define a list of things/suggestions for other people on how to run an event like CSSclasses in their city. This will help us stay aligned and promote the idea even more.

**Comment by Kriesse** [comment]
My priority when growing CSSclasses is to ensure our events are safe + inclusive, while still making it as easy as possible for people to run events. 

I suggest a similar approach we chose at CSSconf.org: Ask people to follow 3 simple rules if they want to use the "CSSclasses brand": http://cssconf.org/

I also think http://nodeschool.io/ is a great example in terms of successfully growing their event series and bringing it to a lot of attendees.


Also moving those notes from Bastian here from an older discussion in #13:

* core values that are not to be messed with to be an "official" CSSclasses
* what do i need (room, coaches)
* how do i coach
* how can i engage people
* how to get help from us

**Comment by KlausTrainer** [comment]
:+1: on the 3 simple rules similar to CSSconf.org.

With regard to Bastian's points on coaching, here my 3 ¢ based on experiences from co-organizing the recent CSSclasses event in Hamburg:

First of all, the [Coaches](http://cssclasses.cssconf.eu/coaches/) section on the website was helpful, and also provided some guidance for organizing the event :)

We addressed the "core values" and "how do i coach" points by pointing coaches to the OpenTechSchool [core values](http://www.opentechschool.org/about.html) as well as the [coaching guide](https://opentechschool.github.io/slides/presentations/coaching/), asking them to read those. 

As one way to providing further assistance and fostering shared understanding, we offered a coaches meeting the day before the workshop. There, we discussed how to apply the core values and coaching guidelines in practice, among other things. The meeting was optional, but I guess that about two thirds of all coaches attended. Based on our own impressions and the feedback we received, they appreciated it and found that it was helpful.

**Comment by Kriesse** [comment]
@KlausTrainer Good points! Would be great to have a section or a separate page about how to organize a coaches training.

**Comment by KlausTrainer** [comment]
@Kriesse Good idea. As soon as I find some time for that, I'll do a draft, and ask for feedback.
